src/main/java/nl/tudelft/ti2206/group9/gui/popup/AbstractInfoPopup.java

```diff
@@ -97,7 +97,9 @@ public Button getRightButton() {
         return right;
     }
 
-    /** Plays the button sound once, given a volumelevel. */
+    /**
+     * Plays the button sound once, given a volumelevel.
+     */
```

Comment:
Yeah, I guessed Maikel can't control his mighty IntelliJ XD I already reverted the most annoying ones (and the checkstyle-warning creating ones) ;p

<review, documentation, layout